Replace unused 'under consideration' position with 'blocked'. (#159)
WebKit/standards-positions: README.md

- `support` - WebKit sees this specification as conceptually good, and worth prototyping, getting feedback on its value, and iterating.
- `neutral` - WebKit does not see this specification as harmful, but is not convinced that it is a good approach or worth working on.
- `oppose` - WebKit considers this specification to be harmful in its current state.
- `not considering` - WebKit does not intend to look at this specification in the near future.

Note that positions on this repository do not reflect implementation status. We might like something we do not get around to implement for one reason or another. And we might dislike something we nevertheless have to implement.

#### Other possible issue states

> - `blocked` - Coming to a position is blocked on issues identified with the spec or proposal.

Reviewer: Maybe:

```suggestion
- `blocked` - assigning a position is blocked on issues identified with the spec or proposal.
```

> - `blocked` - Coming to a position is blocked on issues identified with the spec or proposal.

Reviewer 2: ```suggestion
- `blocked` - Coming to a position is blocked on issues identified with the specification or proposal.
```
(In a prior PR I removed this abbreviation throughout.)

- `duplicate` - The issue is a duplicate of another issue.

> - `invalid` - The issue is not about a specification which would be implemented in a browser engine such as WebKit.

Reviewer: ```suggestion
- `invalid` - The issue is not about a specification that would be implemented in a browser engine such as WebKit.
```
